CHANGELOG.markdown

```diff
@@ -1,3 +1,9 @@
+5.5.3
```

Comment:
`PolyKinds` can be a breaking change under certain circumstances, so go ahead and bump this to 5.6. Similarly, change the "since 5.5.3" Haddock comments to "since 5.6" as well.